### PR TITLE
fix(VProgressLinear): add missing absolute,top,bottom props

### PR DESCRIPTION
**Pull request by stratdev3**
With absolute and bottom props, the VProgressLinear should be align to the bottom border inside a v-toolbar.

Example : 
- correct in v2.x : https://codepen.io/cp10/pen/OJZdBzg?editors=101
- incorrect in v3.0b14 : https://codepen.io/cp10/pen/jOxdeYx?editors=101

This patch fix the regression

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
- [X] The PR title is no longer than 64 characters.
- [X] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [X] My code follows the code style of this project.
- [X] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VProgressLinear/VProgressLinear.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const VProgressLinear = defineComponent({
   name: 'VProgressLinear',
 
   props: {
+    absolute: Boolean,
+    bottom: Boolean,
```
Comment (jacekkarczmarczyk) [inline packages/vuetify/src/components/VProgressLinear/VProgressLinear.tsx:22]: could support `top` as well

Comment (stratdev3) [reply packages/vuetify/src/components/VProgressLinear/VProgressLinear.tsx:22]: add the top property and update documentation

**Comment by KaelWD** [comment]
Other components use the `location` and `position` composables for this

**Comment by stratdev3** [comment]
@KaelWD so what's the plan to fix the issue ?

**Comment by johnleider** [comment]
> @KaelWD so what's the plan to fix the issue ?

He probably means something like this:

```tsx
// located in VAppBar.tsx

location: {
  type: String as PropType<'top' | 'bottom'>,
  default: 'top',
  validator: (value: any) => ['top', 'bottom'].includes(value),
}
```

**Review by johnleider — APPROVED** [review]

**Comment by mediafreakch** [comment]
Thank you for the fix. The documentation is outdated, since the `bottom` prop is no longer supported (progress bar is shown on top, not bottom): https://next.vuetifyjs.com/en/components/progress-linear/#toolbar-loader

Also, eslint-plugin-vuetify complains about the `absolute` prop, even though it seems supported on this component: 

```
<v-progress-linear absolute color="secondary" />
ESLint: 'absolute' has been removed(vuetify/no-deprecated-props)
```

**Pull request merged**